en/flight_modes/mission.md

```diff
@@ -0,0 +1,47 @@
+# MISSION Flight Mode
+
+The MISSION flight mode executes a [predefined mission/flight plan](../flying/missions.md) that has been uploaded to the flight controller. 
+
+> **Note** 
+>  * This mode requires GPS.
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+>  * The vehicle must be armed before this mode can be engaged.
+
+## Description
+
+Missions are usually created in a ground control station (e.g. [QGroundControl](https://docs.qgroundcontrol.com/en/PlanView/PlanView.html)) and uploaded prior to launch. They may also be created by a developer API, and/or uploaded in flight. Individual mission commands are handled in a way that is appropriate for each vehicle's flight characteristics (for example loiter is implemented as *hover* for copter and *circle* for fixed-wing). VTOL vehicles follow the behavior and parameters of fixed-wing when in FW mode, and of copter when in MC mode.
+
+At high level all vehicle types behave in the same way when MISSION mode is engaged:
+
+1. If a mission is stored and PX4 is flying it will execute the [mission/flight plan](../flying/missions.md) from the current step.
+1. If a mission is stored and PX4 is landed:
+   * On copters PX4 will execute the [mission/flight plan](../flying/missions.md). If the mission does not have a `TAKEOFF` command then PX4 will add one, flying the vehicle to the minimum altitude before executing the rest of the flight plan.

```

Comment:
@sanderux This is accurate if landed and at beginning of mission - ie PX4 takes off even if there is no TAKEOFF command and then executes rest of mission. 

What if we're landed half way through a mission, but not disarmed. The user switches from a mode back into mission. What does PX4 do - take off then execute rest of mission? Do nothing?

A minor point of clarification here too. You referred to a takeoff command being added if needed. If you download the mission after it has been added would this TAKEOFF be included in the mission listing? If not then I'd assume that you could simply say that the vehicle takes off (ie not that a command is added).